### PR TITLE
Subsetting

### PR DESCRIPTION
**Pull request by carriewright11**
updating the data to be more environmental

### CODE REVIEW

**Comment by carriewright11** [comment]
ok... want to check with @ehumph  about the source for the data - can we add anything here?

Also I don't like how the gender column isn't more inclusive...typically I would point something out about this or try to avoid that data. There is only one question currently that uses it to get to one answer which isn't so bad... but want to think about this. 

@avahoffman what are your thoughts? In all the examples in the lecture and all the other questions in the lab we ignore that column, which I think helps, but normally I would want to say that two genders is not inclusive. I know we have also worried about making people feel called out as well by pointing that out.

**Comment by carriewright11** [comment]
is there a better clean_names example than this?

CO2 <- 
  read_csv("https://daseh.org/data/Yearly_CO2_Emissions_1000_tonnes.csv")
head(CO2, n = 2)
clean_names(CO2)


Previously used opioid data that had upper and lower case variables

**Comment by ehumph** [comment]
My thoughts on the `gender` variable: this is historical data, and they will likely work with historical data in their job. It seems like this could be a good opportunity to demonstrate how sometimes you get variables with non-inclusive categories, or just variables that aren't great. You could discuss how to approach data analysis with such variables, as well as point out how to identify historical variables that are problematic. My guess is a decent number of them won't even notice that the `gender` variable is non-inclusive until it is pointed out to them, because it's so commonly used with these categories.

**Comment by ehumph** [comment]
> is there a better clean_names example than this?
> 
> CO2 <- read_csv("https://daseh.org/data/Yearly_CO2_Emissions_1000_tonnes.csv") head(CO2, n = 2) clean_names(CO2)
> 
> Previously used opioid data that had upper and lower case variables

All of the datasets had really clean variable names from the start

**Comment by ehumph** [comment]
As for data provenance: it is heat-related ER visits between 2011 and 2022, as reported by the state of Colorado, specifically made available by the Colorado Environmental Public Health Tracking program website. Full dataset available at https://coepht.colorado.gov/heat-related-illness

**Comment by avahoffman** [comment]
I agree, I think a disclaimer is is appropriate for gender. There are pretty progressive folks who use a disclaimer, when working with (as Elizabeth mentioned). Could point to/use the conclusion of this paper's abstract : https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6526522/

Regarding clean names - maybe just pull a random csv from the data folder? If it's brief, no need to use one of the key datasets..

**Comment by carriewright11** [comment]
> I agree, I think a disclaimer is is appropriate for gender. There are pretty progressive folks who use a disclaimer, when working with (as Elizabeth mentioned). Could point to/use the conclusion of this paper's abstract : https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6526522/
> 
> Regarding clean names - maybe just pull a random csv from the data folder? If it's brief, no need to use one of the key datasets..

ok - the one I used makes all the year names into x1891 kind of thing - do we like that?

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd [hunk 1]
```diff
@@ -397,12 +358,14 @@ library(janitor)
...
 
-The `clean_names` function can intuit what fixes you might need. Here it makes everything consistent.
+The `clean_names` function can intuit what fixes you might need. Here it make sure year names aren't just a number, so that the colnames dont need ticks or quotes to be used.
 
 ```{r}
-head(annualDosage, 2)
-clean_AD <- clean_names(annualDosage)
-head(clean_AD, 2)
+CO2 <- 
+  read_csv("https://daseh.org/data/Yearly_CO2_Emissions_1000_tonnes.csv")
```
Comment (avahoffman) [inline modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:365]: why not use dasehr::yearly_co2_emissions ?

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd [hunk 2]
```diff
@@ -439,54 +402,42 @@ clean_names(test)
 This time lets also make it a smaller subset so it is easier for us to see the full dataset as we work through examples. 
 
 ```{r}
-#install.packages(OCSdata)
-#library(OCSdata)
-#OCSdata::load_imported_data("ocs-bp-opioid-rural-urban")
+#read_csv("https://daseh.org/data/Colorado_ER_heat_visits_by_county_gender.csv")
```
Comment (avahoffman) [inline modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:405]: library(dasehr)
CO_heat_ER_bygender

?

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd [hunk 3]
```diff
@@ -695,25 +646,23 @@ https://media.giphy.com/media/5b5OU7aUekfdSAER5I/giphy.gif
...
+🏠 [Class Website](https://daseh.org)    
 💻 [Lab](https://daseh.org/modules/Subsetting_Data_in_R/lab/Subsetting_Data_in_R_Lab.Rmd)
 
 ## Get the data
 
 ```{r}
-#install.packages(OCSdata)
-#library(OCSdata)
-#OCSdata::load_imported_data("ocs-bp-opioid-rural-urban")
+#read_csv("https://daseh.org/data/Colorado_ER_heat_visits_by_county_gender.csv")
```
Comment (avahoffman) [inline modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:655]: library(dasehr)
CO_heat_ER_bygender

?

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd (hunk 1 above)
Comment (carriewright11) [reply modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:365]: you mean that you don't want me to read it in from the csv file? cause that is the dataset I used

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd [hunk 4]
```diff
@@ -90,87 +90,44 @@ We will use a dataset from a project we worked on called Open Case Studies.
...
-library(here)
-annualDosage <- read_csv(file = 
-   here("OCS_data/data/simpler_import/county_annual.csv"))
-
-```
+We will work with data called `ER` about heat-related ER visits between 2011 and 2022, as reported by the state of Colorado, specifically made available by the Colorado Environmental Public Health Tracking program website. Full dataset available at https://coepht.colorado.gov/heat-related-illness.
 
+```{r}
+ER <- 
+  read_csv("https://daseh.org/data/Colorado_ER_heat_visits_by_county_gender.csv")
```
Comment (avahoffman) [inline modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:109]: library(dasehr)
CO_heat_ER_bygender

?

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd (hunk 2 above)
Comment (carriewright11) [reply modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:405]: I will show both  if you want me to show the package- I was concerned we would have people who can't install packages from GitHub like we have had in the past

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd (hunk 3 above)
Comment (carriewright11) [reply modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:655]: sure I will add it

File: modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd (hunk 2 above)
Comment (avahoffman) [reply modules/Subsetting_Data_in_R/Subsetting_Data_in_R.Rmd:405]: I think we should have a preference for the package, but leaving a note that you can also do this makes sense!

**Comment by avahoffman** [comment]
> ok - the one I used makes all the year names into x1891 kind of thing - do we like that?

Yes!!! love this example.

**Comment by carriewright11** [comment]
Gonna merge this!

**Pull request merged**